helm/delivery-varnish/app-configs/delivery-varnish_delivery.yaml

```diff
@@ -6,3 +6,7 @@ service:
 
 elb:
   tags: "systemCode=upp,teamDL=universal.publishing.platform@ft.com,environment=d"
+
+s3:
+  bucketName: content-test-elb-access-logs-eu-west-1
```

Comment:
This config will be applied to team clusters.
Should we disable this for these clusters? Otherwise we should find a solution to write to different buckets per cluster, right?